HealthcareSystem/PatientWebApp/Controllers/AppointmentController.cs

```diff
@@ -72,6 +75,8 @@ public IActionResult PrioritySearchAppointments(AppointmentSearchWithPrioritiesD
                 parameters.InitialParameters.IsAppointmentValid();
                 freeAppointments = _freeAppointmentSearchService.SearchWithPriorities(parameters).ToList();
                 freeAppointments.ForEach(appointment => freeAppointmentsDTOs.Add(ExaminationMapper.ExaminationToExaminationDTO(appointment)));
+                freeAppointmentsDTOs.ForEach(appointmentDTO => appointmentDTO.DoctorName = _doctorService.ViewProfile(appointmentDTO.DoctorJmbg).Name);
+                freeAppointmentsDTOs.ForEach(appointmentDTO => appointmentDTO.DoctorSurname = _doctorService.ViewProfile(appointmentDTO.DoctorJmbg).Surname);
                 return Ok(freeAppointmentsDTOs);
```

Comment:
I really wouldn't call the service method twice, you're making to database queries for the same data here. Just save the Doctor in a temporary variable. Also, why is the method in the DoctorService called ViewProfile? Could it be changed to Get?

Reply:
Fixed in a0b2c915b5a87cfd127459a5fc3f21d216caee18.